addons/mail/static/src/discuss/call/common/thread_actions.js

```diff
@@ -24,11 +24,12 @@ threadActionsRegistry
             component.rtc.toggleCall(component.thread);
         },
         sequence: 10,
-        sequenceQuick: 30,
+        sequenceQuick: (component) => !component.env.inDiscussApp && 30,
```

Comment:
I'm not familiar with `sequenceQuick`, what is the goal of changing existing actions in the context of this PR?